packager/packager.h

```diff
@@ -128,6 +128,11 @@ struct StreamDescriptor {
   std::vector<std::string> dash_accessiblities;
   /// Optional for DASH output. It defines Role elements of the stream.
   std::vector<std::string> dash_roles;
+
+  /// Optional for DASH output. Set to true if only DASH output is needed.
+  bool dash_only = false;
+  /// Optional for HLS output. Set to true if only HLS output is needed.
+  bool hls_only = false;
 };
 
 class SHAKA_EXPORT Packager {
```

Comment:
Can you also update https://github.com/google/shaka-packager/blob/master/docs/source/options/dash_options.rst and https://github.com/google/shaka-packager/blob/master/docs/source/options/hls_options.rst?

It will be great if you can add an example to https://raw.githubusercontent.com/google/shaka-packager/master/docs/source/tutorials/dash_hls_example.rst too?

Reply:
Yes have updated the mentioned files. Please review and let me know.